CMake Error at CMakeLists.txt:240 (include):
  include could not find requested file:

    AwsFindPackage


CMake Error at CMakeLists.txt:243 (aws_use_package):
  Unknown CMake command "aws_use_package".


-- Configuring incomplete, errors occurred!
>>> ERROR: aws-sdk-cpp: build failed

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -232,11 +232,7 @@ if (LEGACY_BUILD)
             set(BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS_PREV})
         endif ()
     else ()
-        # This is required in order to append /lib/cmake to each element in CMAKE_PREFIX_PATH
-        set(AWS_MODULE_DIR "/${CMAKE_INSTALL_LIBDIR}/cmake")
-        string(REPLACE ";" "${AWS_MODULE_DIR};" AWS_MODULE_PATH "${CMAKE_PREFIX_PATH}${AWS_MODULE_DIR}")
-        # Append that generated list to the module search path
-        list(APPEND CMAKE_MODULE_PATH ${AWS_MODULE_PATH})
+        find_package(aws-c-common REQUIRED)
         include(AwsFindPackage)
         set(IN_SOURCE_BUILD OFF)
     endif ()
